### ISSUE TITLE
Verify SSL by default and add configuration option

### ISSUE THREAD

**Issue by vasekch** [cinek810/snow-grafana-proxy#14]
Please make verified SSL request default option, change following line to True:
https://github.com/cinek810/snow-grafana-proxy/blob/38e59f1697ef9964c669a3299c2f9a9b8c0418af/snow-grafana-proxy.py#L135

Introduce new config parameter do disable SSL verification for those, who need it.

New `etc/snow-grafana-proxy.conf`:
```
...
service-now:
        instance:
                url: https://instance.service-now.com
                user: api-user
                password: password
                # verifySSL: no
...
```
I think that yaml parser would allow even values like 'yes', 'no', 1, 0, 'true' and 'false' on top of Python classic 'True', 'False', but needs testing

**Comment by vasekch** [comment]
this is ready here https://github.com/vasekch/snow-grafana-proxy/commit/05e42dbfb2d75252cc5618c7bbe11bf469afdfc5

**Comment by cinek810** [comment]
Merged #18 . Thank you!
